index.html

```diff
@@ -0,0 +1,1209 @@
+
+                  <p>Implementations are encouraged to use mechanisms to record activities about ACL resources for the purpose of accountability and integrity, eg. by having audit trails, notification of changes, reasons for change, preserving provenance information.</p>
+
+                  <p>Implementations that want to allow a class of write or control operations on resources are encouraged to require agents to be authenticated, eg. for provenance records or accountability purposes.</p>
+                </div>
+              </section>
+
+              <section id="privacy-considerations" inlist="" rel="schema:hasPart" resource="#privacy-considerations">
+                <h3 property="schema:name">Privacy Considerations</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p><em>This section is non-normative.</em></p>
+
+                  <p>The class of read and write operations require discrete access permissions:</p>
+
+                  <p>Access permission to append a new resource to a container resource is independently of access permission to read a container resource. Thus, servers are encouraged to prevent information leakage when a successful HTTP request appends a new resource to a container resource. For instance, while the knowledge of the URI-Reference in <code>Location</code> and <code>Content-Location</code> HTTP headers in the response of a <code>POST</code> does not in itself pose a security threat ([<cite><a class="bibref" href="#bib-rfc3986">RFC3986</a></cite>]), servers should consider the risks when read access to the container is not granted to agents.</p>
+
+                  <p>Access permission to update a resource is independently of access permission to read a resource. Thus, servers are encouraged to prevent information leakage when an attempt to delete information in a resource may reveal the existence of the information. For instance, when an HTTP <code>PATCH</code> request uses SPARQL Update’s <code>DELETE DATA</code> operation, servers should consider the risks of disclosing information by the chosen status code when read access to the resource is not granted to agents.</p>
```

Comment:
I think we need to expand on this, it is probably not too easy to understand for someone not already deeply into the debates we've been having... OTOH, those who are implementing this in near future are probably pretty deep into it already, so perhaps there's no urgency.

Reply:
Probably. It is a non-normative security consideration. I tried to keep it somewhat generic because we didn't seem to have rough consensus on the actual behaviour. Some issues: https://github.com/solid/specification/issues/125 , https://github.com/solid/specification/issues/139 , https://github.com/solid/specification/issues/220 :)

Reply:
OK, lets get this detailed later as we go through those.